4.0/test/run

```diff
@@ -54,7 +54,7 @@ prepare() {
 }
 
 run_test_application() {
-  docker run --user=1001 --rm --cidfile=${cid_file} -p ${test_port} ${IMAGE_NAME}-testapp
+  docker run --user=100001 --rm --cidfile=${cid_file} -p ${test_port} ${IMAGE_NAME}-testapp
```

Comment:
We had issues on some systems that this number was too big, could you use just 10001 pls, so we don't need to change config of the testing machines?